.editorconfig

```diff
@@ -3,9 +3,11 @@ root = true
 [*]
 indent_style = space
 indent_size = 2
+tab_width = 2
```

Comment:
Could you please share any examples/styleguides/standards of popular Git repositories which are using `tab` defining indents, so that we could do more research and discussion? 
Otherwise, shall we keep `indent_size=2` as our default JavaScript indent size, and remove this config, which may trick editors which is extended with automatic formatting tools?